Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -40,6 +40,7 @@ open class _ChatChannelListItemView<ExtraData: ExtraDataTypes>: _View, UIConfigP
         .init()
         .withoutAutoresizingMaskConstraints
         .withAdjustingFontForContentSizeCategory
+        .supportingBidirectionalLanguages
```

Comment:
This starting to look like a bit annoying boilerplate but I guess it's ok for now

Reply:
yeah... I think as well. Let's see